IT'S NOT TOO LATE

MuffinOverlord opened the pull request:
TODO:

- [x] Completare la lista delle uni
- [ ] ~~Ordinarle per numero di sottodomini (siti) vulnerabili a SQL Injection~~ Let's not get arrested

Commit:
Start working on a contingency plan

Commit:
Add unibas, unibg, unibo

Commit:
Add unibs, unica

Commit:
Add unical, unicam, unicas

Commit:
Add unict, unicz, unich

MuffinOverlord commented:
@gallorob @beard33 up

Need a feedback on style/formatting/template/whatever

beard33 commented:
L'idea non è male. Appena ho tempo di sistemare bene la cosa te la mergio. L'unica cosa sarebbe interessante aggiungere, dato che è spesso una discriminante, i CFU che si acquisiscono in UNIPV in relazione a quelli richiesti nelle varie università (triennali/magistrali) nei vari ambiti (i.e. INF-08, INF-12 etc)

@MuffinOverlord

ghost commented:
> L'unica cosa sarebbe interessante aggiungere, dato che è spesso una discriminante, i CFU che si acquisiscono in UNIPV in relazione a quelli richiesti nelle varie università (triennali/magistrali) nei vari ambiti (i.e. INF-08, INF-12 etc)

La lista contiene solo i corsi di laurea triennali. Correggimi se sbaglio, ma la questione CFU è valida solo per l'accesso alla magistrale.

Ho fatto un calcolo veloce in caso si volesse affrontare la questione magistrale.

INF/01: 0
ING-INF/05: 63

Nota: Non è possibile aumentare il numero di crediti del settore ING-INF/05, non essendoci esami "A scelta dello studente" di quel settore.
Nota2: I crediti sono aumentati recentemente con il passaggio di "Sistemi Operativi" da 6 a 9 CFU. In futuro potrebbero variare.

Source: http://webing.unipv.eu/wp-content/uploads/2018/06/Regolamento-CL-Ing.-Elettronica-e-Informatica.pdf

beard33 commented:
Sei te che hai cambiato in triennale, non posso saperlo io ahahha. 
Per la magistrale faccio io, ne ho qualcuna da consigliare. Cambia solo il nome al doc MD; l'idea di dare info su altre università non è male, ma in modo _leggermente_ più serio

MuffinOverlord commented:
> Cambia solo il nome al doc MD; 

Il nome è provvisorio. Sono aperto a suggerimenti.

> Per la magistrale faccio io, ne ho qualcuna da consigliare.

Non è meglio essere imparziali ? La lista delle triennali contiene (conterrà) tutte le università d'Italia.

Commit:
Add unife, unifi, unifg, unige

Commit:
Add uninsubria, univaq, unimc, unime

Commit:
Add unimi, unimib, polimi

Commit:
Add unimore, unimol

Commit:
Add unina, unicampania

Commit:
Add uniparthenope, unior, unipd, unipa, unipr, unipg

Commit:
Add uniupo, unipi, univpm

Commit:
lint

Commit:
Add unirc, uniroma1, uniroma2, uniroma3, uniroma4, unisalento, unisa

Commit:
Add unisannio, uniss, unisi

Commit:
Add unite, unito, polito

Commit:
Add unitn, units, unitus, uniud

Commit:
Add uniurb, unive, iuav, univr

Commit:
To be fair, Unife supports https

Kind of.
Your connection is downgraded to plain http but the https request is accepted by the server

Commit:
unimi joined the party

(They didn't, another redirect from https to http)

ghost commented:
honorable mentions:

```
Università degli Studi della BASILICATA
Università degli Studi dell'AQUILA
Università degli Studi di NAPOLI "L'Orientale"
Università degli Studi di ROMA "Foro Italico"
Università degli Studi ROMA TRE
Università IUAV di VENEZIA

For the lack of SSL/TLS (HTTPS) support
```



```
Università degli Studi di BOLOGNA
Università degli Studi della Campania "Luigi Vanvitelli"
Università degli Studi di Urbino Carlo Bo
Università Politecnica delle MARCHE

For the best SSL/TLS (HTTPS) implementation
```

Bonus:
```
Università degli Studi di PALERMO

For being the only one with a DNS Certification Authority Authorization (CAA) policy
(If you had HSTS you would have won the best SSL/TLS (HTTPS) implementation)
```

beard33 committed:
Rename SAVEYOURSELF.md to ListaUniversitàItaliane.md

beard33 commented:
Ti ho dato una rinominata al file, per il resto gg (non so con che sbatti tu l'abbia fatto)

Dopo faccio i conti esatti dei crediti e li aggiungo in calce
